src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -743,4 +743,11 @@
   <data name="BadBuiltInOps3" xml:space="preserve">
     <value>'{0}' was not a valid built-in operator name</value>
   </data>
+  <data name="HintNameInvalidSegment" xml:space="preserve">
+    <value>The hintName '{0}' contains an invalid segment '{1}' at position {2}.</value>
+    <comment>{0}: the provided hintname. {1}: the invalid segment, {2} the position it occurred at</comment>
+  </data>
+  <data name="HintNameNotRelative" xml:space="preserve">
+    <value>The hintName '{0}' is not relative.</value>
```

Comment:
>The hintName '{0}' is not relative. [](http://example.com/codeflow?start=11&length=35)

Is this message reported for a path such as `../..`? (Arguably, that path is relative.) If so, perhaps the message should be "... within the current directory" or something similar.